core/templates/dev/head/pages/exploration_editor/simple_editor_tab/ScrollSyncService.js

```diff
@@ -87,16 +88,9 @@ oppia.factory('ScrollSyncService', [
       var documentY = $document.scrollTop();
       for (var i = 0, minDy; i < targets.length; i++) {
```

Comment:
Optional: we don't really use this pattern (minDy) elsewhere in the codebase; consider pulling it out into its separate var statement above this one. That said, this is optional.